source/cl_main.c

```diff
@@ -872,28 +873,38 @@ void CL_RelinkEntities (void)
 
 		if (ent->effects & EF_RAYGREEN)
 		{
+#ifdef __WII__
+			QMB_RocketTrail(oldorg, ent->origin, RAYGREEN_TRAIL);
```

Comment:
Doesnt `R_RocketTrail` hook into `QMB_RocketTrail` on PSP and CTR? can you confirm and make Wii follow this same structure so we can get rid of this ifdef